docs/subnets/cross-subnet-communication.md

```diff
@@ -0,0 +1,72 @@
+# Cross-Subnet Communication
+
+Avalanche Warp Messaging (AWM) enables native cross-Subnet communication and allows [Virtual Machine
+(VM)](./README.md#virtual-machines) developers to utilize any-to-any communication between the
+validators of two Subnets. The content of the messages can contain arbitrary payloads. 
```

Comment:
```
any-to-any communication between the validators of two Subnets
```
This reads to be like any validator in Subnet A can send a message to any validator in Subnet B.

This really isn't the point of Warp (sending messages between validators like this is trivial)... Warp is intended to enable any Subnet A to be able to send a message to any other Subnet B. Generally Subnet A (as an entity) sending a message to Subnet B (as an entity). This means that every validator in Subnet B should receive this message.

```suggestion
Avalanche Warp Messaging (AWM) enables native cross-Subnet communication and allows [Virtual Machine
(VM)](./README.md#virtual-machines) developers to implement arbitrary communication protocols between any two Subnets.
```